CHANGELOG.md

```diff
@@ -7,122 +7,65 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
-- TransactionListItem: fix save/cancel rounded btn ([#14659](https://github.com/MetaMask/metamask-extension/pull/14659))
-- Add New Tokens Found page ([#14366](https://github.com/MetaMask/metamask-extension/pull/14366))
-- Swaps / STX improvements ([#14622](https://github.com/MetaMask/metamask-extension/pull/14622))
-- Simplify removeSnap functionality after snaps-skunkworks update ([#14646](https://github.com/MetaMask/metamask-extension/pull/14646))
-- Fixing Speed up/Cancel information text ([#14551](https://github.com/MetaMask/metamask-extension/pull/14551))
-- Fix SRP paste duplication ([#14625](https://github.com/MetaMask/metamask-extension/pull/14625))
-- LavaMoat - UI upgrade - secure package naming ([#14565](https://github.com/MetaMask/metamask-extension/pull/14565))
-- E2e send tests ([#14568](https://github.com/MetaMask/metamask-extension/pull/14568))
-- slip44@2.1.0 ([#14619](https://github.com/MetaMask/metamask-extension/pull/14619))
-- Reproducible `.zip` files ([#14623](https://github.com/MetaMask/metamask-extension/pull/14623))
-- fix incorrect casing of includeUsdRate option passed to CurrencyRateController ([#14615](https://github.com/MetaMask/metamask-extension/pull/14615))
-- Fix redundant entry in lockfile ([#14616](https://github.com/MetaMask/metamask-extension/pull/14616))
-- Jest tests - incremental coverage ([#14612](https://github.com/MetaMask/metamask-extension/pull/14612))
-- build - update bify-module-groups for build determinism ([#14610](https://github.com/MetaMask/metamask-extension/pull/14610))
+- Add Coinbase Pay as fiat onramp option ([#14648](https://github.com/MetaMask/metamask-extension/pull/14648))
 - Add search information for Theme dropdown ([#14476](https://github.com/MetaMask/metamask-extension/pull/14476))
-- Update version parsing to allow rollback release ([#14288](https://github.com/MetaMask/metamask-extension/pull/14288))
-- Update bug-report.yml ([#14466](https://github.com/MetaMask/metamask-extension/pull/14466))
-- Disable STX if a regular tx is in progress ([#14554](https://github.com/MetaMask/metamask-extension/pull/14554))
-- Fix development build scripts ([#14594](https://github.com/MetaMask/metamask-extension/pull/14594))
-- Update PULL_REQUEST_TEMPLATE.md ([#14547](https://github.com/MetaMask/metamask-extension/pull/14547))
-- Patch gridplus-sdk for perf fix ([#14582](https://github.com/MetaMask/metamask-extension/pull/14582))
-- Add applyLavaMoat build flag ([#14583](https://github.com/MetaMask/metamask-extension/pull/14583))
-- fix cross-fetch moderate vulnerability alert ([#14570](https://github.com/MetaMask/metamask-extension/pull/14570))
-- Fix browser.tab.get call ([#14550](https://github.com/MetaMask/metamask-extension/pull/14550))
-- Added getAccounts suppressUnauthorized param ([#14126](https://github.com/MetaMask/metamask-extension/pull/14126))
+- **[FLASK]** Add in-app notifications for snaps ([#14605](https://github.com/MetaMask/metamask-extension/pull/14605))
```

Comment:
Is "snap"/"snaps" something we want to capitalize? I've noticed this in a few translations where we capitalize Snaps. @FrederikBolding

Reply:
good question

however, this need not be a blocker for the changelog, and we can update this later, so I am going to merge before this question in resolved

Reply:
Good question @digiwand - I'll check with the team to get our shared opinion on it!